src/components/tabs.js

```diff
@@ -46,6 +47,13 @@
       }
     }, [isDev, currentTab]);
 
+    useEffect(() => {
+      // Manually trigger updateIndicator() to show active tab on page load (TECHSUP-8534)
+      setTimeout(() => {
```

Comment:
Why the timeout? wont a check like` if(tabsRef.current)` suffice?

Reply:
@JorisPannekeet Please see the research I conducted in [TECHSUP-8534](https://bettyblocks.atlassian.net/browse/TECHSUP-8534?focusedCommentId=205814). Other implementations simply won't work.

Reply:
@RobD-tech I see, in your research however i do not see the following implementation:
`useEffect(()=>{
 if(tabsRef.current){ // Check if the element is rendered
  // do logic
}
},[tabsRef.current]); 
`

Can you validate if that works? If not then the timeout is fine.

Reply:
@JorisPannekeet Unfortunately your implementation also doesn't work.

Reply:
Alright, thanks for checking. I've approved the mr.

Reply:
Why timeout 100? Why not 200? Or 50? Unless you have an explanation for that, it's not wise to use a solution like this.

Reply:
@jeroenreijs I have tested with various values and the fix becomes consistent with a value of 100 in the use cases that I've tested. You don't want this value to be too large because you don't want the end user to notice the delay, i.e. see the indicator appearing some time after the page has already loaded. The underlying reason why a timeout is needed to fix this bug is because the active tab indicator includes an animated transition which apparently influences it's appearance.